tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -349,6 +349,10 @@ protected override bool Skip (Type type, string protocolName)
 				case "OSLogEntryLog":
 				case "OSLogEntrySignpost":
 					return true;
+				// iOS 16
+				case "DDDevice":
```

Comment:
Is this the right test ignore, these are new types, but some of this protocol ignores are for protocols that were added afterward, right?

Reply:
Intro tests were failing with the following error messages

```
Introspection.iOSApiProtocolTest
[FAIL] DDDevice conforms to NSCoding but does not implement INSCoding
[FAIL] DDDeviceEvent conforms to NSCoding but does not implement INSCoding
```

and the header is defined like:

```objc
DD_EXTERN
API_AVAILABLE( ios( 16.0 ) )
@interface DDDevice : NSObject
// ...
@end
```

so it seems that Apple is using the protocol internally. Is this the right way to ignore them?